upup/pkg/fi/cloudup/awstasks/elastic_ip.go

```diff
@@ -229,6 +231,10 @@ func (_ *ElasticIP) RenderAWS(t *awsup.AWSAPITarget, a, e, changes *ElasticIP) e
 	} else {
 		publicIp = a.PublicIP
 		eipId = a.ID
+		err := t.AddAWSTags(*a.ID, changes.Tags)
```

Comment:
We really should set the tags on create also, even though it isn't used yet, because at some stage someone is going to set some tags and be mighty confused :-)